inc/functions.php

```diff
@@ -49,6 +49,20 @@ function core_sitemaps_get_sitemaps() {
 	return $core_sitemaps->registry->get_sitemaps();
 }
 
+/**
+ * Retrieves the prefix for the sitemap filename.
+ *
+ * @return string Prefix.
+ */
+function core_sitemaps_filename_prefix() {
```

Comment:
`filename` might be misleading because they're not actually files. Maybe just `core_sitemaps_prefix`? Or perhaps `core_sitemaps_sitemap_prefix`

Reply:
`core_sitemaps` will be removed when merging into core!?! If so, I would prefer `core_sitemaps_sitemap_prefix`.

Reply:
That hasn't been decided yet. We might also just replace `core_sitemaps` with `wp_sitemaps` or end up using something like `wp_sitemap_prefix` in the end. But this doesn't matter for the sake of adding it to the plugin 🙂